pkg/monitoring/rules/alerts/operator.go

```diff
@@ -75,10 +75,10 @@ func operatorAlerts() []promv1.Rule {
 		},
 		{
 			Alert: "VMStorageClassWarning",
-			Expr:  intstr.FromString("(count(kubevirt_ssp_vm_rbd_block_volume_without_rxbounce > 0) or vector(0)) > 0"),
+			Expr:  intstr.FromString("(count(kubevirt_ssp_vm_rbd_block_volume_without_rxbounce * on(name, namespace) (kubevirt_vmi_info{guest_os_name=\"Microsoft Windows\"} > 0 or kubevirt_vmi_info{os=~\"windows.*\"} > 0) > 0) or vector(0)) > 0"),
```

Comment:
@machadovilaca Are you sure that guest_os_name will always be "Microsoft Windows" for windows and no abbreviation?